api/src/main/java/org/pmiops/workbench/api/CloudTaskEgressController.java

```diff
@@ -3,13 +3,16 @@
 import org.pmiops.workbench.exfiltration.EgressRemediationService;
 import org.pmiops.workbench.model.ProcessEgressEventRequest;
 import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Qualifier;
 import org.springframework.http.ResponseEntity;
 import org.springframework.web.bind.annotation.RestController;
 
 @RestController
 public class CloudTaskEgressController implements CloudTaskEgressApiDelegate {
 
-  @Autowired private EgressRemediationService egressRemediationService;
+  @Autowired
+  @Qualifier("sumologic-egress-service")
```

Comment:
I saw few others naming conversions https://github.com/all-of-us/workbench/search?q=Qualifier
Worth to keep them consitent, e.g.  static variable and use sumologicEgressService

Reply:
thanks will update it